migrations/V183__Add_MTO_Common_Solution.sql

```diff
@@ -1,26 +1,37 @@
 CREATE TYPE MTO_COMMON_SOLUTION_KEY AS ENUM (
-    'SOLUTION_A',
-    'SOLUTION_B'
+    'INNOVATION', 'ACO_OS', 'APPS', 'CDX', 'CCW', 'CMS_BOX', 'CMS_QUALTRICS', 'CBOSC', 'CPI_VETTING', 'EFT', 'EXISTING_CMS_DATA_AND_PROCESS', 'EDFR', 'GOVDELIVERY', 'GS', 'HDR', 'HPMS', 'HIGLAS', 'IPC', 'IDR', 'LDG', 'LV', 'MDM_POR', 'MARX', 'OUTLOOK_MAILBOX', 'QV', 'RMADA', 'ARS', 'CONNECT', 'LOI', 'POST_PORTAL', 'RFA', 'SHARED_SYSTEMS', 'BCDA', 'ISP', 'MIDS', 'MDM_NCBP', 'MODEL_SPACE'
 );
-
 CREATE TYPE MTO_SOLUTION_TYPE AS ENUM (
     'IT_SYSTEM',
     'CONTRACT',
     'CROSS_CUTTING_GROUP',
     'OTHER'
 );
+
+CREATE TYPE MTO_COMMON_SOLUTION_SUBJECT AS ENUM (
+    'APPLICATIONS_AND_PARTICIPANT_INTERACTION_ACO_AND_KIDNEY_MODELS',
+    'APPLICATIONS_AND_PARTICIPANT_INTERACTION_NON_ACO_MODELS',
+    'MEDICARE_FEE_FOR_SERVICE',
+    'CONTRACT_VEHICLES',
+    'LEARNING',
+    'QUALITY',
+    'COMMUNICATION_TOOLS_AND_HELP_DESK',
+    'MEDICARE_ADVANTAGE_AND_PART_D',
+    'PAYMENTS_AND_FINANCIALS',
+    'DATA',
+    'LEGAL'
+);
+
 COMMENT ON TYPE MTO_SOLUTION_TYPE IS

```

Comment:
thanks! Yes, I implemented and expanded comments. Thanks for noting, I wasn't thinking about this since I was primarily adding entries. I went ahead and added comments to the milestone table as well in the spirit of this.